hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/Federation.md

```diff
@@ -161,34 +161,39 @@ Configuration
 
 These are common configurations that should appear in the **conf/yarn-site.xml** at each machine in the federation.
 
+| Property                          | Example                  | Description                                                                 |
+|:----------------------------------|:-------------------------|:----------------------------------------------------------------------------|
+| `yarn.federation.enabled`         | `true`                   | Whether federation is enabled or not                                        |
+| `yarn.resourcemanager.cluster-id` | `<unique-subcluster-id>` | The unique subcluster identifier for this RM (same as the one used for HA). |
 
-| Property | Example | Description |
-|:---- |:---- |:---- |
-|`yarn.federation.enabled` | `true` | Whether federation is enabled or not |
-|`yarn.resourcemanager.cluster-id` | `<unique-subcluster-id>` | The unique subcluster identifier for this RM (same as the one used for HA). |
-
-#### State-Store:
+#### How to configure State-Store
 
 Currently, we support ZooKeeper and SQL based implementations of the state-store.
 
 **Note:** The State-Store implementation must always be overwritten with one of the below.
 
 ZooKeeper: one must set the ZooKeeper settings for Hadoop:
 
-| Property | Example | Description |
-|:---- |:---- |:---- |
-|`yarn.federation.state-store.class` | `org.apache.hadoop.yarn.server.federation.store.impl.ZookeeperFederationStateStore` | The type of state-store to use. |
-|`hadoop.zk.address` | `host:port` | The address for the ZooKeeper ensemble. |
+| Property                            | Example                                                                             | Description                             |
+|:------------------------------------|:------------------------------------------------------------------------------------|:----------------------------------------|
+| `yarn.federation.state-store.class` | `org.apache.hadoop.yarn.server.federation.store.impl.ZookeeperFederationStateStore` | The type of state-store to use.         |
+| `hadoop.zk.address`                 | `host:port`                                                                         | The address for the ZooKeeper ensemble. |
 
```

Comment:
While writing the Markdown document, I noticed that the tables were not properly aligned. I have aligned the tables to make the content more readable when we edit it.